pkg/costmodel/allocation.go

```diff
@@ -1665,8 +1670,13 @@ func applyUnmountedPVs(window kubecost.Window, podMap map[podKey]*Pod, pvMap map
 		podMap[key].Allocations[container].Properties.Namespace = namespace
 		podMap[key].Allocations[container].Properties.Pod = pod
 		podMap[key].Allocations[container].Properties.Container = container
-		podMap[key].Allocations[container].PVByteHours = unmountedPVBytes[cluster] * window.Minutes() / 60.0
-		podMap[key].Allocations[container].PVCost = amount
+		unmountedBreakDown := map[string]kubecost.PVUsage{
+			kubecost.UnmountedSuffix: {
+				ByteHours: unmountedPVBytes[cluster] * window.Minutes() / 60.0,
+				Cost:      amount,
+			},
+		}
+		podMap[key].Allocations[container].AddPVBreakDown(unmountedBreakDown)
```

Comment:
@nikovacevic I am unsure about this update that was necessary because I removed PVCost and PVByteHours

Reply:
See comment about this function